components/PartsList/index.js

```diff
@@ -3,31 +3,64 @@ import {
   PartsListContainer,
   StyledHeading,
   FilterButtonContainer,
+  FilterContainer,
 } from "./PartsList.styled";
 import { useState } from "react";
 
 export default function PartsList({ parts }) {
   const [categoryFilter, setCategoryFilter] = useState("alle");
+  const [statusFilter, setStatusFilter] = useState("alle");
 
   const filteredParts = parts.filter((part) => {
-    if (categoryFilter === "alle" || part.category === categoryFilter)
-      return true;
+    if (categoryFilter === "alle" || part.category === categoryFilter) {
+      if (statusFilter === "alle") {
+        return part;
```

Comment:
in `part`you are storing all your parts  I guess. 
the naming should be accordingly: `parts`

Reply:
oh. nevermind, I didnt see the filter above. :) all good, `part`is coorect here.

Reply:
not sure if i get you right, but i am using filter method on 'parts' and returning each filtered 'part' into filteredParts